.github/workflows/operator.yaml

```diff
@@ -47,26 +73,139 @@ jobs:
+        uses: actions/checkout@v4
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: temurin
+          cache: maven
+
+      - name: Show config
+        run: |
+          cd operator
+          make config-show
+
+      - name: Build
+        run: |
+          cd operator
+          make BUILD_OPTS="--no-transfer-progress" SKIP_TESTS=true build
+
+      - name: Login to quay.io registry
+        run: |
+          docker login -u "${{ secrets.QUAY_USERNAME }}" -p "${{ secrets.QUAY_PASSWORD }}" quay.io
+
+      - name: Build and publish operator image
+        run: |
+          cd operator
+          make image-build image-push
+
+      - name: Build and publish operator bundle # TODO: Also push to DockerHub Registry
+        run: |
+          cd operator
+          make bundle
+
+      - name: Build and publish operator catalog # TODO: Also push to DockerHub Registry
+        run: |
+          cd operator
+          make catalog
+
+
+  operator-update-install-file:
+    name: Update Operator Install file
+    runs-on: ubuntu-latest
+    needs: operator-publish
+    steps:
+
+      - name: Checkout ${{ github.ref_name }}
+        run: |
+          mkdir registry
+          cd registry
+          git init
+          git config --global user.name "apicurio-ci"
+          git config --global user.email "apicurio.ci@gmail.com"
+          git remote add origin "https://apicurio-ci:${{ secrets.ACCESS_TOKEN }}@github.com/Apicurio/apicurio-registry.git"
+          git fetch
+          git checkout ${{ github.ref_name }}
+          git branch --set-upstream-to=origin/${{ github.ref_name }}
+          git pull
 
       - name: Set up JDK 17
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v4
         with:
           java-version: '17'
-          distribution: 'temurin'
-          cache: 'maven'
+          distribution: temurin
+          cache: maven
 
-      - name: Workaround jackson-coreutils
+      - name: Show config
         run: |
-          # upstream issue: https://github.com/java-json-tools/jackson-coreutils/issues/59
-          rm -rf ~/.m2/repository/com/github/java-json-tools
-          mkdir -p /tmp/coreutils-workaround
-          ( cd /tmp/coreutils-workaround && mvn dependency:get -DremoteRepositories=https://repo1.maven.org/maven2 -Dartifact=com.github.java-json-tools:jackson-coreutils:2.0 )
+          cd registry/operator
+          make config-show
 
-      - name: Run the tests in local mode
-        run: ./mvnw clean verify -P '!external_repos' -DskipOperatorTests=false -pl operator/controller -am
+      - name: Build
+        run: |
+          cd registry/operator
+          make BUILD_OPTS="--no-transfer-progress" SKIP_TESTS=true build
+
+      - name: Update install file
+        run: |
+          cd registry/operator
+          make INSTALL_FILE=install/install.yaml dist-install-file
```

Comment:
It can for sure, but it shouldn't, the CI is already pretty heavy, and rebuilding the Operator just to run a diff is not worth it in my opinion.

Happy to ask another pair of eyes in case is necessary.

Reply:
Done.